app/src/androidTestMock/java/com/example/android/architecture/blueprints/todoapp/tasks/TasksActivityTest.kt

```diff
@@ -63,6 +70,10 @@ class TasksActivityTest {
 
     private lateinit var repository: TasksRepository
 
+    @get:Rule
+    val composeTestRule = createAndroidComposeRule<TasksActivity>()
+    private val activity by lazy { composeTestRule.activity }
```

Comment:
I guess it doesn't really matter but wouldn't `get()=` be technically more correct? Assuming you are using this just to delay the reference to activity

Reply:
I guess both could work, I don't have a strong preference there, I'm just keeping it consistent with the other Compose samples we have out there

Reply:
Sure! Done